.github/workflows/draft-release.yml

```diff
@@ -20,14 +20,12 @@ jobs:
 
       - name: install and build
         run: pnpm install --frozen-lockfile
-      # TODO: Reuse step for GA release
-      # - uses: sap/cloud-sdk-js/.github/actions/get-changelog@main
-      #   name: Get Changelog
-      #   id: get-changelog
+      - uses: sap/cloud-sdk-js/.github/actions/merge-changelogs@main
```

Comment:
[q] Just for my understanding, we don't call this in the bump action because we dont want to maintain a root CHANGELOG.md?